src/crystal/system/system.cr

```diff
@@ -0,0 +1,24 @@
+# :nodoc
+module Crystal
+  # :nodoc
+  module System
+    # :nodoc
+    module System
```

Comment:
I think that the methods in this module could go in just `Crysatal::System` itself.